src/Castle.Core/DynamicProxy/Generators/MetaMethod.cs

```diff
@@ -52,6 +52,8 @@ public string Name
 			get { return name; }
 		}
 
+		public bool Ignore { get; internal set; }
+
 		public bool Proxyable { get; private set; }
 
 		public bool Standalone { get; private set; }
```

Comment:
As an aside unrelated to this PR, we have multiple `bool` flags in `MetaMethod`. I haven't checked how the runtime lays this type out in memory, but I'm assuming /hoping that the flags are packed efficiently. If they ever turn out to blow up this type's size in memory, we can always combine them into a single flags enum.